.github/actions/generate-system-logs/action.yml

```diff
@@ -0,0 +1,32 @@
+name: ""
+description: "Generate system logs for hardware debugging"
+
+inputs:
+  run_args:
+    description: 'Commands to run in docker'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Get Runner Host Name
+      id: get_hostname
+      shell: bash
+      run: |
+        echo "HOSTNAME=$(hostname)" >> $GITHUB_ENV
+        echo "TIMESTAMP=$(date +'%Y%m%d%H%M%S')" >> $GITHUB_ENV
+    - name: Determine Docker Tag to use
+      shell: bash
+      run: |
+        rm -rf ~/run-log
```

Comment:
do you have to remove? Is it because it's homedir?

Why is it homedir btw? So it's persistent?

Reply:
I remove to introduce a soft cap on file storage usage even though its only a few MB at most.
Homedir was for persistence.

Reply:
I guess in case it doesn't make it to the upload artifact step ? That would be the persistence in my mind